configure_wrapper

```diff
@@ -0,0 +1,4 @@
+#! /bin/sh
+cp conftest.py bak_conftest.py
+./real_configure $@
```

Comment:
Wouldn't it be safer to change `$@` to `"$@"` to avoid arguments with spaces being broken up (not sure if it's portable, at least it works in bash)

also does the `real_configure` script support running it in a different directory? if yes then it may be a good idea to also support it by changing it to `"$(dirname "$0")"/real_configure`

Reply:
`real_configure` is just a copy of `configure`.
I never heard about running `configure` in a different directory.